### PR TITLE
Editorial changes for grammar and formatting.

### PR DESCRIPTION
**Pull request by MCamp859**
Completed text cleanup referenced in #881  
Signed-off-by: MCamp859 <maryx.camp@intel.com>

### CODE REVIEW

File: source/get-started/virtual-machine-install/virt-manager.rst [hunk 1]
```diff
@@ -155,51 +148,47 @@ This will launch the `New VM` dialog box, shown in figure 4 below.
...
 
       Figure 10: New VM Ready to begin the installation dialog box
 
-#. Clear Linux requires an UEFI firmware to boot and operate correctly so you
-   need to customize the installation process by changing the firmware from
-   `BIOS` to `UEFI x86_64`. In the `Firmware` field, select the
-   `UEFI x86_64: /usr/share/qemu/OVMF.fd` entry as shown in figure 11 and 
-   select the :guilabel:`Apply` button.
+#. Customize the installation process by changing the firmware from `BIOS` to
+   `UEFI x86_64`. In the `Firmware` field, select the :file:`UEFI
```
Comment (mvincerx) [inline source/get-started/virtual-machine-install/virt-manager.rst]: After `UEFI x86_64`, please add (portion of original):
"|CL| requires UEFI firmware."

UEFI firmware is almost universal with Linux distros, but it's worth noting here.

File: source/get-started/virtual-machine-install/virt-manager.rst [hunk 2]
```diff
@@ -15,116 +15,110 @@ These instructions support the |CL| live-server installer to create the |CL|
...
+   .. figure:: figures/virtmgr/virt-manager-03.png
+      :scale: 100%
+      :alt: Create a new virtual network
 
-   Figure 3: Create a new virtual network
+      Figure 3: Create a new virtual network
 
-To accept the default values as shown in the figure above, select the 
-:guilabel:`Finish` button to create your virtual network.
+   To accept the default values, select the :guilabel:`Finish` button.
```
[the remark this answers is not in the document]
Comment (MCamp859) [reply source/get-started/virtual-machine-install/virt-manager.rst]: Whoops, this was not supposed to float below the figure, I moved it up with the other text.

**Comment by MCamp859** [comment]
@mvincerx  Please look at the latest commit and see what you think.

**Comment by MCamp859** [comment]
@intelkevinputnam  In lines 110-130, I noticed that nested # steps repeat 1,2,3 instead of switching to a, b,c. Is that something we can add to the template To Do list or is it a known issue?

File: source/get-started/virtual-machine-install/virt-manager.rst [hunk 3]
```diff
@@ -15,125 +15,118 @@ These instructions support the |CL| live-server installer to create the |CL|
...
 
-#. This will bring you to step 2 of the wizard where you can choose ISO or 
-   CDROM install media. 
+#. In step 2 of the `New VM` wizard, you can choose ISO or CDROM install
+   media.
 
-   #. Before browsing for the ISO image, un-check the 
-      `Automatically detect from the installation media / source` field and
-      then select the :guilabel:`Browse...` button as shown in figure 5
+   a. Uncheck `Automatically detect from the installation media / source`
```
Comment (mvincerx) [inline source/get-started/virtual-machine-install/virt-manager.rst]: @MCamp859. Here's a simple way to fix a nested auto-enumerated ordered list.

**Comment by mvincerx** [comment]
LGTM. Thanks, @MCamp859

**Comment by mvincerx** [comment]
Closes #881

**Pull request merged**